packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts

```diff
@@ -43,6 +45,9 @@ async function compileFixture({ input, dirname }: { input: string; dirname: stri
                 ],
             }),
         ],
+        onwarn(warning) {
+            warnings.push(warning);
```

Comment:
I would recommend checking whether or not the `warning` is related to `lwc:dynamic` before silencing it. Swallowing all the errors might hide some unexpected behavior in tests.